src/sentry/mail/actions.py

```diff
@@ -48,12 +49,16 @@ def after(self, event, state):
         if features.has(
             "organizations:issue-alert-fallback-targeting", self.project.organization, actor=None
         ):
-            fallthrough_type = FallthroughChoiceType(self.data["fallthroughType"])
+            fallthrough_choice = self.data.get("fallthroughType", None)
```

Comment:
If you wanted to keep the type to just string, you could do `self.data.get("fallthroughType", "")`
I haven't looked it would make sense.